tensorboard/webapp/widgets/line_chart_v2/lib/drawable.ts

```diff
@@ -23,15 +23,26 @@ import {
 import {PaintBrush} from './paint_brush';
 import {ObjectRenderer} from './renderer/renderer_types';
 
-class RenderCacheContainer {
-  private prevFrameCache = new Map<string, any>();
-  private currFrameCache = new Map<string, any>();
+type Cacheable = {};
 
-  getFromPreviousFrame(key: string): any {
-    return this.prevFrameCache.get(key);
+export interface RenderCachePublicApi {
+  getFromPreviousFrame(key: string): Cacheable | null;
+  setToCurrentFrame(key: string, value: Cacheable): void;
+}
+
+class RenderCacheContainer implements RenderCachePublicApi {
+  private prevFrameCache = new Map<string, Cacheable>();
+  private currFrameCache = new Map<string, Cacheable>();
+
+  getFromPreviousFrame(key: string): Cacheable | null {
+    const value = this.prevFrameCache.get(key);
+    return value ?? null;
   }
 
-  setToCurrentFrame(key: string, value: any) {
+  setToCurrentFrame(key: string, value: Cacheable) {
+    if (!value) {
+      console.log(key);
```

Comment:
Did you intend to keep this here? If so, could you make the message a bit clearer?

Reply:
Nope, it was for my own debugging. Remove it. Thanks for catching it